app/src/main/java/ch/admin/bag/dp3t/reports/ReportsFragment.java

```diff
@@ -129,12 +132,26 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
 			if (tracingStatusInterface.isReportedAsInfected()) {
 				headerType = ReportsHeaderFragment.Type.POSITIVE_TESTED;
 				infectedView.setVisibility(View.VISIBLE);
+
+				// Show the onset date of the report
+				long onsetDateInMillis = secureStorage.getPositiveReportOnsetDate();
+				if (onsetDateInMillis > 0L) {
+					infectedView.findViewById(R.id.card_encounters_faq_who_is_notified_container).setVisibility(View.VISIBLE);
+					String formattedDate = DateUtils.getFormattedDateWrittenMonth(onsetDateInMillis);
+					String faqText = getString(R.string.meldungen_positive_tested_faq2_text).replace("{ONSET_DATE}", formattedDate);
+					Spannable formattedText = StringUtil.makePartiallyBold(faqText, formattedDate);
+					((TextView) infectedView.findViewById(R.id.card_encounters_faq_who_is_notified)).setText(formattedText);
+				} else {
+					infectedView.findViewById(R.id.card_encounters_faq_who_is_notified_container).setVisibility(View.GONE);
+				}
+
 				infectedView.findViewById(R.id.delete_reports).setOnClickListener(v -> {
 					AlertDialog.Builder builder = new AlertDialog.Builder(requireContext(), R.style.NextStep_AlertDialogStyle);
 					builder.setMessage(R.string.delete_infection_dialog)
 							.setPositiveButton(R.string.delete_infection_dialog_finish_button, (dialog, id) -> {
 								tracingStatusInterface.resetInfectionStatus(getContext());
 								secureStorage.setIsolationEndDialogTimestamp(-1L);
+								secureStorage.setPositiveReportOnsetDate(-1L);
```

Comment:
This should also be done in the HomeFragment.showEndIsolationDialogIfNecessary() => to prevent errors like that in the future, we should share to code...